### ISSUE TITLE
non-prepared statements mode?

### ISSUE THREAD

**Issue by grypyrg** [akopytov/sysbench#126]
It seems that all new `.lua` scripts use prepared statements to execute statements.

I had to fall back to using the `oltp_legacy` scripts in order to get what I want (I was going busy trying to make a demo for `sys.statements_analysis`, but that does not show prepared statements at all)(related 'bug': https://bugs.mysql.com/bug.php?id=73112)

As in the dangerous real world many applications do not use prepared statements, I think having support for regular statements would be needed in the `sysbench` `.lua` scripts.

Can the lua mysql driver be set to 'client-side' prepared statements so not a lot of code rewrite would be necessary?

**Comment by akopytov** [comment]
Hi Kenny,

Cannot agree more on all points, and the answer to your question is `--db-ps-mode=disable` (weird syntax, I know, will be fixed in 1.1).

This is likely going to be a frequently asked question, will include into the upcoming FAQ section in the manual.

**Comment by grypyrg** [comment]
Thanks!

Found it now in `help`:
```
General database options:

  --db-driver=STRING  specifies database driver to use ('help' to get list of available drivers)
  --db-ps-mode=STRING prepared statements usage mode {auto, disable} [auto]
  --db-debug[=on|off] print database-specific debug information [off]
```
